src/main/java/com/powsybl/openloadflow/ac/outerloop/ReactiveLimitsOuterLoop.java

```diff
@@ -207,45 +215,91 @@ private static boolean switchPqPv(List<PqToPvBus> pqToPvBuses, ContextData conte
      *  - if Q equals to Qmin
```

Comment:
This comment should be updated :
```suggestion
     * (1) A bus PV bus can be switched to PQ in 3 cases:
     *  - if Q equals to Qmax
     *  - if Q equals to Qmin
     *  - if the controller bus is a remote controller bus and V is outside realistic voltage limits
```